src/scss/_glaze.scss

```diff
@@ -0,0 +1,35 @@
+// =================================
+// Glaze Color Scheme
+//
+// This mixin should be optionally called to override
+// the base JE colour palette variables with Glaze
+// specific colours
+//
+// N.b. Variables defined inside this mixin need to be applied
+// with !global to override its equivalent in the base palette
+// (due to Sass scoping)
+// =================================
+
+@mixin applyScheme-glaze {
```

Comment:
Just out of interest – what is Glaze? (I think it's the design system update, but I'm not sure!)

If so, I'm guessing the long term plan is to make this the default colour theme?

Reply:
@ashleynolan Yeah, it is a redesign but currently for apps only, that's why we need a mechanism to apply it conditionally. But likely will be on web soon and then we can make it a default theme.